README.md

```diff
@@ -1,3 +1,7 @@
-# Repository to maintain our dockerfiles
+# Dockerfiles
 
-Currently, this repository will maintain our Dockerfile's. In the future the goal is to push base images to dockerhub, so that we can save some time instead of re-building everything all the time.
+Repository for a collection of `Dockerfile`s that have helped us during development and during projects. Useful for local development, deployments and CI/CD pipelines.
+
+Currently, we only offer public `python-poetry` images and files, where `poetry` is the amazing [python package manager](https://python-poetry.org/). So far other images we found were outdated and unmaintained. We wanted to change that, as we believe in `poetry` as the future oof python package management.
```

Comment:
Typo: 
```suggestion
Currently, we only offer public `python-poetry` images and files, where `poetry` is the amazing [python package manager](https://python-poetry.org/). So far other images we found were outdated and unmaintained. We wanted to change that, as we believe in `poetry` as the future of python package management.
```

Reply:
Maybe folder should also be named `python-poetry`? (:

Reply:
Hm, what do you mean by that? The current structure `python/3.8/poetry` creates a pretty meaningful path, I believe. 

I adapted it from public `Docker` repos, like [the official python one](https://github.com/docker-library/python), just with an extra level for `python` itself so that we can support multiple technologies in the future.

Reply:
> The current structure `python/3.8/poetry` creates a pretty meaningful path, I believe.

Oh, I didn't get that we are expecting to add something else rather then poetry to the python folder. While reading the `python-poetry` in the readme, I thought the structure is: `python-poetry/3.8`

Reply:
For now we'll just have poetry, in the future we may also have just a python base with some additions and call it something different.